test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -78,19 +79,20 @@ const (
 )
 
 const (
-	configFile               = "config/performance-config.yaml"
-	extensionPointsLabelName = "extension_point"
+	configFile                               = "config/performance-config.yaml"
+	extensionPointsLabelName                 = "extension_point"
+	schedulerSubsystemAttemptDurationSeconds = metrics.SchedulerSubsystem + "_" + metrics.SchedulingAttemptDurationSeconds
 )
 
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
 		Metrics: map[string]*labelValues{
-			"scheduler_framework_extension_point_duration_seconds": {
```

Comment:
FYI: `scheduler_framework_extension_point_duration_seconds`, `scheduler_scheduling_attempt_duration_seconds` and `scheduler_pod_scheduling_duration_seconds` are three different names.